frontend/ui/network/manager.lua

```diff
@@ -624,7 +634,8 @@ end
 function NetworkMgr:getPowersaveMenuTable()
     return {
         text = _("Disable Wi-Fi connection when inactive"),
-        help_text = _([[This will automatically turn Wi-Fi off after a generous period of network inactivity, without disrupting workflows that require a network connection, so you can just keep reading without worrying about battery drain.]]),
+        help_text = Device:isKindle() and _([[This is unlikely to function properly on a stock Kindle, given how chatty the framework is.]]) or
```

Comment:
It works just fine for those of us that run frameworkless :)

Reply:
Not over here (the numbers above were from a frameworkless session) ;o).

It's an old device that has almost never been online, which probably plays a part here, but even after letting it sit connected in the framework, I still get deltas of ~360.

Reply:
I have iptables rules that only allows outgoing connections on the local lan (i dont need any helpful connections by amazon) so  that is why it works for me

Reply:
Fyi in case you have any other suggestions. https://hosted.weblate.org/translate/koreader/koreader/en/?checksum=4f0eae549f1bd665#comments

> The English can probably be improved but "talkative, communicative..." is indeed correct. :-) It means the Kindle framework is constantly communicating with Amazon servers or something along those lines.
>
> In Dutch I provisionally translated it as "sends/broadcasts much information," which would likely be clearer in English too.

![image](https://github.com/koreader/koreader/assets/202757/dcb58f44-f58b-4ae6-90c9-b8390419d2fb)

Reply:
Yeah, it's probably a bit jargony. I can't quite think of something else as succinct, though; so the alternative would be to explain it, i.e., the the native system already triggers a lot of network activity on its own.

Reply:
Yes, that's probably clearer. "given how much network activity the framework generates" or something.